x-pack/test/functional/apps/maps/group3/reports/index.ts

```diff
@@ -64,7 +74,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       await PageObjects.reporting.clickGenerateReportButton();
 
       const percentDiff = await measurePngDifference('example_map_report');
-      expect(percentDiff).to.be.lessThan(0.09);
+      expect(percentDiff).to.be.lessThan(0.01);
```

Comment:
I notice the latest baseline image for the `PNG file matches the baseline image, using embeddable example` test is not showing the 3 triangles that we had in the previous baseline.

Reply:
I am not sure why the 3 triangles are not displaying.

Reply:
https://github.com/elastic/kibana/issues/156623